packages/suite/src/actions/wallet/formDraftActions.ts

```diff
@@ -0,0 +1,55 @@
+import { GetState } from '@suite-types';
+import { getFormDraftKey } from '@wallet-utils/formDraftUtils';
+import { FormDraftKeyPrefix, FormDraft } from '@wallet-types/formDraft';
```

Comment:
```suggestion
import type { FormDraftKeyPrefix, FormDraft } from '@wallet-types/formDraft';
```
i'm aware that somewhere in previous PR i asked you to not using them, but then we agreed at some point to use `import type` annotation. sorry for confusion.

Reply:
Ahaaa :) I was hoping you'll get to this point. Thank you.